app/.env.example

```diff
@@ -3,7 +3,6 @@ VUE_APP_PATH="/"
 #vaults web-url
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
 VUE_APP_AUTH_WEB_URL="https://dev.account.bcregistry.gov.bc.ca/"
-VUE_APP_BUSINESSES_URL="https://dev.account.bcregistry.gov.bc.ca/"
```

Comment:
This is the old key for the MBR (which is now BRD and is no longer in Auth Web).